src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -117,10 +128,19 @@ class MetadataBasedItemList extends React.Component<Props, State> {
                 break;
             default: {
                 const data = getProp(item, 'metadata.data', {});
-                const mdFieldName = this.getMetadataColumnName(
-                    metadataColumnsToShow[columnIndex - FIXED_COLUMNS_NUMBER],
+                const mdFieldName = this.getMetadataColumnName(metadataColumn);
```

Comment:
Could we get the cell data in one shot using `getProp` and string interpolation with the `mFieldName`?

Reply:
As discussed on Slack, `mdFieldName` is column name. The same function is used to get header value so returning cell data from the function using `getProp` is not useful here.